packages/calling-stateful-client/review/calling-stateful-client.api.md

```diff
@@ -173,7 +175,7 @@ export interface RemoteVideoStreamState {
 
 // @public
 export interface StatefulCallClient extends CallClient {
-    createView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState, options?: CreateViewOptions): Promise<void>;
```

Comment:
Is this a breaking change? Should I do `Promise<void | VideoStreamRendererAndView` instead?

Reply:
You are good: https://www.typescriptlang.org/play?#code/PTAEFcDsBMFMDMCWlbVIgzqAhhjiBzSbAIwBtZQAXAe1ADcbE0TwrRIb2qALSrvgCdQAd2wBPHIJpRoAOgBQIUAGU6AYx7ZIBZARyh8RbFXCD+8A-CjqqiGpGpb24DKmp1zbM48bNQALQAfKDmpoKOsgjI7phKYNigJNjqANZigtBY6jQAtgAO2IhUZCSUmtoEsIoKOZAY7OYYNGT0sAAqNABqTGgAvDgY4pDqoAAUAJQAXKAACtK5mLAAPH7QIX0hAN4AvgDctQ4NobDNrR00AKow0Sj9g8PqkzPzeUvLUUh3G9sKoP8ncKRG5fVAHfYKQ71RqwIHtcT5WAAQTwsEEdiOoAGuEe4wmWN+ANAFHY2BmawORMS2LExROZzanR6zEmlIB1JwtJhDIu1zgoOgrMhRJJSRmnxi0DZ-xIWM5RW5LUZ3V6QqJsppCvpSt5IMlrNACh2kNF2EEBHazgAogAPRG2VDM6DwxEzSYEuYLd5rIIHM0W6122AO6BOl2UAZNHVM3p+82Wky2+1UR29cNyqPnTp825gk2wqQBxNBkM5gXht34zaet5uD5675xotUJPBlPQMuS9OR07RlXMJsJlsl9udu7d7VZq4NvNAA